src/lib.rs

```diff
@@ -333,7 +333,7 @@ impl LsColors {
     }
 
     /// Check if an indicator has an associated color.
-    fn has_color_for(&self, indicator: Indicator) -> bool {
+    pub fn has_color_for(&self, indicator: Indicator) -> bool {
```

Comment:
I think this makes sense but maybe we should rename the method to something like `has_explicit_style_for()`

Reply:
Ok, I renamed it